crate/user-feedback.md

```diff
@@ -1,17 +1,25 @@
 ## Issues
 
-If you have any problems with or questions about this image, please contact us through a [GitHub issue](https://github.com/crate/docker-crate/issues).
-
-If you have any questions or suggestions, we are happy to help! Feel free to join our [public Crate community on Slack](https://crate.io/docs/support/slackin/).
-
-For further information and official contact visit [https://crate.io](https://crate.io).
+Please report issues via the [GitHub issue
+tracker](https://github.com/crate/docker-crate/issues).
 
 ## Contributing
 
-You are very welcome to contribute features or fixes! Before we can accept any pull requests to Crate Data we need you to agree to our [CLA](https://crate.io/community/contribute/). For further information please refer to [CONTRIBUTING.rst](https://github.com/crate/crate/blob/master/CONTRIBUTING.rst).
+This image is primarily maintained by [Crate.io](http://crate.io/), but we
+welcome community contributions!
+
+See the [developer
+docs](https://github.com/crate/docker-crateblob/master/DEVELOP.rst) and the
+[contribution
+docs](https://github.com/crate/docker-crate/blob/master/CONTRIBUTING.rst)
+for more information.
 
-## Documentation
+## Help
 
-Documentation for this image is stored in the [`%%REPO%%/` directory](https://github.com/docker-library/docs/tree/master/%%REPO%%) of the [`docker-library/docs` GitHub repo](https://github.com/docker-library/docs). Be sure to familiarize yourself with the [repository's `REAMDE.md` file](https://github.com/docker-library/docs/blob/master/README.md) before attempting a pull request.
+Looking for more help?
 
-Visit [Crate on Docker](https://crate.io/docs/install/containers/docker/) and get further documentation about how to get started with Crate.
+- Read [the project documentation](https://crate.io/docs/)
+- Check [StackOverflow](https://stackoverflow.com/tags/crate) for common
+  problems
+- Chat with us on [Slack](https://crate.io/docs/support/slackin/)
+- Get [paid support](https://crate.io/pricing/)
```

Comment:
This entire `user-feedback.md` file was removed over in https://github.com/docker-library/docs/pull/888, especially to streamline the long content (similar in intent to this PR).  If you want to customize the content in that new "Quick reference" section, you probably want to modify `crate/get-help.md` to point to these other resources (see any of `.template-helpers/get-help.md`, `ibmjava/get-help.md`, `neo4j/get-help.md`, `oraclelinux/get-help.md`, `silverpeas/get-help.md`, `solr/get-help.md`, `websphere-liberty/get-help.md`, or `xwiki/get-help.md` for examples).